uvm/hier/usr/share/untangle/bin/ut-backup.sh

```diff
@@ -46,6 +46,13 @@ function backupSettings()
     # only match specific versions without the date/version info so we don't backup old files
     # use -L so symlinks are dereferenced
     find /usr/share/untangle/settings/ \( -type f -o -type l \) -regextype sed ! -regex '.*/.*-version-[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}-[0-9\.]*\.js' -exec cp -L --parents {} $temp/ \;
+
+    # Copy branding image if present
+    VAR_IMAGES_DIR=/var/www/images
+    BRANDING_LOGO_FILE=$VAR_IMAGES_DIR/BrandingLogo.png
+    if [ -f "$BRANDING_LOGO_FILE" ]; then
+        cp $BRANDING_LOGO_FILE $temp/usr/share/untangle/settings/
```

Comment:
After creating the tarball, remove the branding logo from settings.

Reply:
Since we are copying the logo file under `$temp/usr/share/untangle/settings` directory and removing the entire `$temp` on line #61, I think we don't need to cleanup anything.
Please correct if I am wrong.

Reply:
Oh, my mistake.  I thoguht we were copying to the live /usr/share/untangle/settings.  Nevermind.